src/pages/ContactUsPage/ContactUsForm.tsx

```diff
@@ -27,18 +27,23 @@ const ContactUsForm = () => {
     };
 
     return (
-      <div className="flex flex-col mr-[104px] gap-8 ">
-        <div className="text-lg w-[520px] font-light">
+      <div className="flex flex-col pr-[104px] gap-8">
+        <div className="text-lg font-light lg:w-[513px] md:w-[513px] xs:w-[295px]">
```

Comment:
let's make sure we have no other choice with width here, this should be the last option we can use